docs/en/user_guides/dataset_prepare.md

```diff
@@ -110,7 +110,25 @@ Download Waymo open dataset V1.2 [HERE](https://waymo.com/open/download/) and it
 python tools/create_data.py waymo --root-path ./data/waymo/ --out-dir ./data/waymo/ --workers 128 --extra-tag waymo
 ```
 
-Note that if your local disk does not have enough space for saving converted data, you can change the `out-dir` to anywhere else. Just remember to create folders and prepare data there in advance and link them back to `data/waymo/kitti_format` after the data conversion.
+Note that:
+
+- If your local disk does not have enough space for saving converted data, you can change the `out-dir` to anywhere else. Just remember to create folders and prepare data there in advance and link them back to `data/waymo/kitti_format` after the data conversion.
+
+- If you want faster evaluation on Waymo, you can download the preprocessed [metainfo](<>) containing `contextname` and `timestamp` to the directory `data/waymo/waymo_format/`. Then, the dataset config is modified like the following:
+
+  ```python
+  val_evaluator = dict(
+      type='WaymoMetric',
+      ann_file='./data/waymo/kitti_format/waymo_infos_val.pkl',
+      waymo_bin_file='./data/waymo/waymo_format/gt.bin',
+      data_root='./data/waymo/waymo_format',
+      file_client_args=file_client_args,
+      convert_kitti_format=True,
+      idx2metainfo='data/waymo/waymo_format/idx2metainfo.pkl'
+      )
+  ```
+
+  Experimently, this will reduce evalaution time by around 6 min. Now, this trick is only used in methods based in point cloud.
```

Comment:
The whole evaluation process on Waymo costs around 6 min with 8 A100 GPUs? We need to specify the equipment.

Reply:
Now this trick is only used in point cloud detection model

Reply:
> The whole evaluation process on Waymo costs around 6 min with 8 A100 GPUs? We need to specify the equipment.

It reduces time **by** 6 min. But the file reading is not related to GPU and it's related to file IO speed, CPUs and other staffs. So I ignore it.